src/Avalonia.Base/Threading/DispatcherFrame.cs

```diff
@@ -120,6 +120,13 @@ internal void Run(IControlledDispatcherImpl impl)
                 Dispatcher.RequestProcessing();
                 impl.RunLoop(_cancellationTokenSource.Token);
             }
+            catch (Exception ex)
+            {
+                if (!Dispatcher.TryHandleUnhandledException(ex))
+                {
+                    throw;
+                }
+            }
```

Comment:
Is there a reason to wrap dispatcher frame, when we already wrap dispatcher operation?

Reply:
Note, that it doesn't match WPF behavior.
See https://github.com/dotnet/wpf/blob/main/src/Microsoft.DotNet.Wpf/src/WindowsBase/System/Windows/Threading/Dispatcher.cs#L2101-L2150